### PR TITLE
Step 42 and 43

### PR DESCRIPTION
**Pull request by Soleyu**
Added steps 42 and 43. Also added a punctuation that was missing in 42 and added two more translations in tutorial guide

Fixed the deal with the commits. Just deleted my fork and made it again

### CODE REVIEW

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml [hunk 1]
```diff
@@ -9,6 +9,13 @@
 
 &lt;COL ffdc78&gt;また、ジョブレベルだけではなく装備品もレベルに
 見合っていることが前提です&lt;/COL&gt;</original>
+    <translation>All quests have a &lt;COL ffdc78&gt;"Recommended Level"&lt;/COL&gt;
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml]: Missing period.

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml [hunk 2]
```diff
@@ -9,6 +9,13 @@
...
+This will tell you level of enemies that appear in the quest area.
+
+&lt;COL ffdc78&gt;Note that the recommended level not only
+means your Vocation level but also your Equipment level.&lt;/COL&gt;
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml]: Vocation and equipment probably don't need to be capitalized.

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml [hunk 3]
```diff
@@ -18,5 +25,11 @@
 
 装備品がレベルに見合わない場合もクリア難度は
 上がるため、挑む前には準備も整えましょう</original>
+    <translation>In "World Quests" and "Main Quests"
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step43.xml]: Probably should have a comma at the end.

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml [hunk 4]
```diff
@@ -11,6 +11,17 @@
 
 【&lt;KC UI_CHAT&gt;】を押してチャットモードに
 切替え、発言を入力しましょう</original>
+    <translation>&lt;COL ffdc78&gt;"Chat"》&lt;/COL&gt;
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml]: "》".

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml [hunk 5]
```diff
@@ -24,6 +35,12 @@
 &lt;COL ffdc78&gt;拡大した状態であれば、発言者を直接選んで
 話しかけたり、プロフィール確認やパーティー要請、
 フレンド申請できます&lt;/COL&gt;</original>
+    <translation>The Information that is displayed can be filtered
+with [&lt;KC UI_LB&gt;／&lt;KC UI_RB&gt;] this will switch channels.
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml]: "／" should be changed to the regular "/".

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml [hunk 6]
```diff
@@ -24,6 +35,12 @@
...
+Use (+) to add a channel.
+
+In the input field press [&lt;KC UI_RT&gt;] to enlarge the log
+Use [&lt;KC KEY_UP&gt;／&lt;KC KEY_DOWN&gt;] to scroll through it.</translation>
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml]: "／" should be changed to the regular "/".

File: ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml [hunk 7]
```diff
@@ -48,6 +71,15 @@
...
 　&lt;COL ffdc78&gt;Group&lt;/COL&gt; ：グループチャットメンバー
 　&lt;COL ffdc78&gt;Entry Board&lt;/COL&gt; ：エントリーボード参加者</original>
+    <translation>The chat targets are as follows:
+
+　&lt;COL ffdc78&gt;Say&lt;/COL&gt;: Surrounding players.
+　&lt;COL ffdc78&gt;Shout&lt;/COL&gt;: All players in the world.
+　&lt;COL ffdc78&gt;Party&lt;/COL&gt;: Party members.
+　&lt;COL ffdc78&gt;Tell&lt;/COL&gt;: Specific player.
+　&lt;COL ffdc78&gt;Clan&lt;/COL&gt;: Clan members
+　&lt;COL ffdc78&gt;Group&lt;/COL&gt; ：グループチャットメンバー
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/steps/tutorial_guide_step46.xml]: This line is left undone?

File: ui/00_message/tutorial_guide/tutorial_guide.xml [hunk 8]
```diff
@@ -920,6 +921,7 @@
   <message>
     <key>TUTORIAL_365</key>
     <original>ボーナスダンジョン</original>
+    <translation>Bonus Dungeon</translation>
```
Comment (riftcrystal) [inline ui/00_message/tutorial_guide/tutorial_guide.xml]: Please remove this or make it plural (I have a local change for this that will conflict if left as-is).

**Review by riftcrystal — CHANGES_REQUESTED** [review]

File: ui/00_message/tutorial_guide/tutorial_guide.xml (hunk 8 above)
Comment (Soleyu) [reply ui/00_message/tutorial_guide/tutorial_guide.xml]: will remove this then

**Review by riftcrystal — APPROVED** [review]

**Pull request merged**